domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/OpphørsdatoForUtbetalinger.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.su.se.bakover.domain.revurdering
+
+import no.nav.su.se.bakover.common.startOfMonth
+import no.nav.su.se.bakover.domain.avkorting.Avkortingsvarsel
+import java.time.LocalDate
+
+@JvmInline
+value class OpphørsdatoForUtbetalinger private constructor(
+    private val opphørsdato: LocalDate,
```

Comment:
```suggestion
    val value: LocalDate,
```
Tror jeg stemmer for dette over get() :shrug: Men skulle helst extenda LocalDate